app/models/concerns/rubygem_searchable.rb

```diff
@@ -37,9 +40,13 @@ def as_indexed_json(_options = {})
              }
 
     mapping do
-      indexes :name, analyzer: 'rubygem'
-      indexes :indexed, type: 'boolean'
-      indexes :info, analyzer: 'english'
+      indexes :name, type: 'multi_field' do
+        indexes :name, analyzer: 'rubygem'
+        indexes :suggest, type: 'completion'
```

Comment:
The `completion` type is more suited to the auto-complete feature. If we want to first add the "Did you mean?" functionality, we should be able to just use a regular analyzer (eg. `simple`, as in the [example](https://github.com/elastic/elasticsearch-rails/blob/master/elasticsearch-rails/lib/rails/templates/searchable.dsl.rb#L17), and then use the [`suggest`](https://github.com/elastic/elasticsearch-rails/blob/master/elasticsearch-rails/lib/rails/templates/searchable.dsl.rb#L209) DSL method in the query. I have to play with it with some real data, to see how it behaves.